frontend/src/lib/i18n/en.json

```diff
@@ -37,6 +37,7 @@
     "email_placeholder": "example@email.com",
     "password_placeholder": "E.g. CP23-174a-f4%&-aHAB",
     "sign_in": "Sign in",
+    "sign_in_title": "Sign in",
```

Comment:
I was thinking that it may not make sense in all languages to have same text in the sign in button and in the page title. For example in finnish I would probably prefer "Sisäänkirjautuminen" as the page title.

Reply:
Ok, that makes sense but it's not really apparent from the code. Perhaps the other i18n key could be renamed to match its use case: eg. sign_in_button or sign_in_command. A comment would also be nice but we might have to leave that out because there are no comments in json. Also, we need to decide whether we use camel case or snake case in i18n. Currently we are using both, which is not ideal.